var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -99,7 +99,15 @@ def cmake_args(self):
             self.define_from_variant('ZFP_WITH_ALIGNED_ALLOC', 'aligned'),
             self.define_from_variant('ZFP_WITH_CACHE_TWOWAY', 'twoway'),
             self.define_from_variant('ZFP_WITH_CACHE_FAST_HASH', 'fasthash'),
-            self.define_from_variant('ZFP_WITH_CACHE_PROFILE', 'profile')
+            self.define_from_variant('ZFP_WITH_CACHE_PROFILE', 'profile'),
+            self.define_from_variant('ZFP_WITH_CUDA', 'cuda')
```

Comment:
Can we move the cuda variant up to just under the fortran variant?  I'd like to keep the advanced variants separate at the bottom.

Reply:
Done.